ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java

```diff
@@ -42,7 +42,7 @@
             "Affixes", "PreviousTag1", "PreviousTag2", "PreviousTagPatternLevel1",
             "PreviousTagPatternLevel2", "AggregateContext", "AggregateGazetteerMatches",
             "PrevTagsForContext", "PredictionsLevel1", "GazetteersFeatures", "WordEmbeddings",
-            "BrownClusterPaths", "Linkability"};
```

Comment:
If there is a dependency on Wikifier we may have a problem... does adding the flag result in any exceptions/problems if the source for these features is not present?